src/pt/sussyscan/src/eu/kanade/tachiyomi/extension/pt/sussyscan/SussyToons.kt

```diff
@@ -11,7 +11,7 @@ class SussyToons : GreenShit(
 ) {
     override val id = 6963507464339951166
 
-    override val versionId = 2
+    override val versionId = 3
```

Comment:
did they change the manga urls/ids? since bumping this `versionId` forces all users to migrate back to the same source to use it, as old mangas in library are invalidated